rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @person = person
+    @book = book
+  end
+
+  def add_rentals(_book, date)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] Kindly remove `add_rentals` method from this class as that is not what the previous reviewer meant. You should keep the constructor as it was.

  ```
  def initialize(date, book, person)
    @date = date
    @person = person
    @book = book

    book.rentals << self
    person.rentals << self
  end
  ```